docs/vta/install.md

```diff
@@ -267,6 +297,72 @@ We recommend setting the `VTA_HW_COMP_THREADS` variable in the Makefile to take
 
 Once the compilation completes, the generated bitstream can be found under `<tvm root>/vta/build/hardware/xilinx/vivado/<configuration>/export/vta.bit`.
 
+### Chisel-based Custom VTA Bitstream Compilation for DE10-Nano
+
+Similar to the HLS-based design, high-level hardware parameters in Chisel-based design are listed in the VTA configuration file [Configs.scala](https://github.com/dmlc/tvm/blob/master/vta/hardware/chisel/src/main/scala/core/Configs.scala), and they can be customized by the user.
+
+For Intel FPGA, bitstream generation is driven by a top-level `Makefile` under `<tvmroot>/vta/hardware/intel`.
+
+If you just want to generate the Chisel-based VTA IP core for the DE10-Nano board without compiling the design for the FPGA hardware, enter:
+```bash
+cd <tvmroot>/vta/hardware/intel
+make ip
+```
+Then you'll be able to locate the generated verilog file at `<tvmroot>/vta/build/hardware/intel/chisel/<configuration>/VTA.DefaultDe10Config.v`.
+
+If you would like to run the full hardware compilation for the `de10-nano` board:
```

Comment:
edit this string `de10nano`

Reply:
This has been revised, see the changes I've made [here](https://github.com/dmlc/tvm/pull/3494/files#diff-97dae209acba8bb338938a382f1a3d00).